config/routes.rb

```diff
@@ -2,6 +2,7 @@
   
    get 'personal_page/index'
    get 'fun_page/index'
+   get 'social_projects_page/index'
```

Comment:
Yes, I would say that a social project in your application is an object you're working with. Therefore, with Rails you would usually use resources in the routes file to work with those.

The syntax you're using is usually for something like a completely custom isolated route or something that can't be mapped to an object your system is working on. 

So it's more of a convention you should use in Rails.